llarp/router/rc_lookup_handler.cpp

```diff
@@ -248,7 +248,7 @@ namespace llarp
 
     if(useWhitelist)
     {
-      static constexpr size_t LookupPerTick = 25;
+      static constexpr size_t LookupPerTick = 5;
```

Comment:
Rename to "LookupPerCall" or something (since we don't call this every tick now).

Reply:
Scratch that.  Instead let's change this logic to still call this explore function every tick but do nothing on 4 out of 5 ticks if a service node:

```C++
int _explore_tick = 0; // (member variable)
```

```C++
if (isSvcNode) {
  // Service nodes explore 1 node every 5 ticks
  if (++_explore_tick < 5)
    return;
  else
    _explore_tick = 0;
}
// else client: explore 1 node every tick
```